src/main/java/sb/persistence/dao/OrderDAO.java

```diff
@@ -0,0 +1,94 @@
+package sb.persistence.dao;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import sb.domain.entity.Order;
+import sb.domain.entity.User;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class OrderDAO {
+    
+    private NamedParameterJdbcTemplate namedJdbcTemplate;
+    private BeanPropertyRowMapper<Order> rowMapper;
+
+    public OrderDAO(NamedParameterJdbcTemplate namedJdbcTemplate) {
+        this.namedJdbcTemplate = namedJdbcTemplate;
+        rowMapper = new BeanPropertyRowMapper<>(Order.class);
+    }
+
+    private final String SQL_SELECT_ALL = "select id, status, submitted_by, submitted_at, updated_at " +
```

Comment:
You should not return all the records from db. Add pagination here.

Reply:
fix: add pagination. in case of order i guess all records are needed, do not remove anything here, but in case of supplier remove payer_number, registration_certificate_number